SabreTools.DatFiles/Formats/SoftwareList.cs

```diff
@@ -577,6 +584,7 @@ protected override ItemType[] GetSupportedTypes()
                 ItemType.DipSwitch,
                 ItemType.Disk,
                 ItemType.Info,
+                ItemType.DataArea,
```

Comment:
This may or may not be doing what you think it's doing. Did you look at how the `Rom` items are being written out?